src/Resources/UserResource.php

```diff
@@ -106,6 +117,36 @@ public static function getModel(): string
         return config('filament-user-resource.model');
     }
 
+    public static function navigationLabel(?string $string): void
+    {
+        static::$workNavigationLabel = $string;
+    }
+
+    public static function pluralLabel(?string $string): void
+    {
+        static::$workPluralLabel = $string;
+    }
+
+    public static function label(?string $string): void
+    {
+        static::$workLabel = $string;
+    }
+
+    protected static function getNavigationLabel(): string
```

Comment:
I don't think we need to overwrite these methods - we can just set the properties on the parent class instead.